src/clj/witan/send/model/output.clj

```diff
@@ -48,15 +48,16 @@
 
 (defn prep-ribbon-plot-data
   [data years colours]
-  (let [n-years (count years)
-        filter-data (map #(pull-year data %) (range n-years))
-        ay (first (first filter-data))
-        uppers (create-CI-map "upper" filter-data 1 n-years)
-        lowers (create-CI-map "lower" filter-data 2 n-years)
-        non-zero-data (filter valid-years-vector? (first data))
-        min-year (first (first non-zero-data))
-        max-year (first (last non-zero-data))]
-    (merge lowers uppers {:year ay})))
+  (if (not (nil? (first data)))
```

Comment:
I think this could be `(when (first data)` unless there is a second leg I don't see. You can also nil pun directly.